test/dedup-id/src/test/test.js

```diff
@@ -0,0 +1,213 @@
+'use strict';
+
+const moment = require('abacus-moment');
+const oauth = require('abacus-oauth');
+const request = require('abacus-request');
+const { extend } = require('underscore');
+const util = require('util');
+const uuid = require('uuid');
+
+const debug = require('abacus-debug')('abacus-dedup-id-test');
+
+const doGet = util.promisify(request.get);
+const doPost = util.promisify(request.post);
+
+const authServerURL = process.env.AUTH_SERVER_URL || 'http://localhost:9882';
+const collectorURL = process.env.COLLECTOR_URL || 'http://localhost:9080';
+const reportingURL = process.env.REPORTING_URL || 'http://localhost:9088';
+const systemClientId = process.env.CLIENT_ID;
+const systemClientSecret = process.env.CLIENT_SECRET;
+const objectStorageClientId = process.env.OBJECT_STORAGE_CLIENT_ID;
```

Comment:
What is this `objectStorage` client? Is this the user's credentials? Why not use the system credentials for the test? Is there any value in using a resource-specific client / scopes?

Reply:
If there is some reason to use such a client, I would probably name the variables `providerClientID` and `providerClientSecret`.